app/controllers/user_books_controller.rb

```diff
@@ -1,12 +1,18 @@
 class UserBooksController < ApplicationController
 
   def index
-    books = UserBooksFacade.all_books(params[:auth_token])
+    books = UserBooksFacade.all_books(params[:auth_token])   # needs a user id and a filter_by(can be default nothing) in the params
```

Comment:
I _think_ Lawrence mentioned that the auth_token includes user information we need to hit the Google Books API - what are you planning to use the user_id info for?